### DISCUSSION TITLE
How to remove the shadow after a dropdown out of active status.

### DISCUSSION THREAD

**Discussion by DevinJohw** [shadcn-ui/ui#3849] [Q&A]
I am new to shancn-ui, I used a dropdown component in my next.js app today. I noticed that the dropdown have a shadow when it is out of active status. I was wondering how to remove the shadow.
![2024-05-28 23 20 56](https://github.com/shadcn-ui/ui/assets/143488669/c526afc0-2150-45bf-be6c-83581e71576f)

**Comment by alvinsjoy** [comment]
I asked this question in the Next.js Discord community and got a [solution](https://nextjs-forum.com/post/1242180940537659503).
Here is the [link](https://discord.com/channels/752553802359505017/1242180940537659503) to Discord post. If you have any queries you can reopen the post on Discord.
Hope this helps.
